core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -794,7 +794,7 @@ object IO {
    * Returns a action that will never produce anything. The moral
    * equivalent of `while(true) {}`, only without the wasted CPU cycles.
    */
-  final def never[A]: IO[Nothing, A] = Never.asInstanceOf[IO[Nothing, A]]
+  final def never: IO[Nothing, Nothing] = Never.asInstanceOf[IO[Nothing, Nothing]]
```

Comment:
This can be turned into a value, I think:

```scala
val never: IO[Nothing, Nothing] = 
  IO.async { (k: (ExitResult[Nothing, Nothing]) => Unit) => } }
```

Reply:
Also the old `Never` can be deleted.